spec/features/recipes_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.feature 'Recipes management', type: :feature do
+  let(:user) { create(:user) }
+
+  before do
+    login_as(user, scope: :user)
+    visit recipes_path
+  end
+
+  scenario 'User views recipe index page' do
+    expect(page).to have_content('Find your favourite recipe here')
+    expect(page).to have_link('Add a Recipe')
+  end
+
```

Comment:
- [ ] This test is failing because upon opening that page, the content you are looking for won't show until you add a recipe and that is why it is failing, so you will have to mock adding of the data too before the expect.
![image](https://github.com/fatmahussein/Recipe-app/assets/86200356/fd6033bd-07f6-467f-8448-29d5c9eed60c)